src/test/java/apicurito/tests/utils/openshift/OpenShiftWaitUtils.java

```diff
@@ -11,9 +12,8 @@
 import java.util.function.Predicate;
 import java.util.function.Supplier;
 
-import io.fabric8.kubernetes.api.model.Pod;
-import io.fabric8.kubernetes.api.model.PodCondition;
-import lombok.extern.slf4j.Slf4j;
+import static org.assertj.core.api.Assertions.fail;
+import static org.assertj.core.api.Java6Assertions.assertThat;
```

Comment:
maybe wanted to use `org.assertj.core.api.Assertions.assertThat`?

Reply:
changed